### ISSUE TITLE
Specifying font color for `valueBox()`.

### ISSUE THREAD

**Issue by kputschko** [rstudio/flexdashboard#409]
I notice that some of my value boxes are now defaulting to a black font color instead of the white that it used to be.  Looking inside the function definition, I see there is some work being done to automatically pick a font color based on the background color using

```
color <- htmltools::parseCssColors(color)
    colorText <- getColorContrast(color)
    colorIcon <- sassValue(paste0("mix(", color, ",", getColorContrast(colorText), 
      ",50%)"))
```

I'm wondering if we could simply control the font color ourselves to override this default behavior?

**Comment by cpsievert** [comment]
Yes, that seems like a good idea. Would you like to make a pull request?
